boards/matek/h743-wlite/nuttx-config/include/board.h

```diff
@@ -0,0 +1,493 @@
+/************************************************************************************
+ * nuttx-configs/px4_fmu-v6u/include/board.h
+ *
+ *   Copyright (C) 2016-2019 Gregory Nutt. All rights reserved.
```

Comment:
@dakejahl Is it weird to have the old copyright on this file? I mean if it is derivative it needs it, but shouldn't it include additional changes

Reply:
I don't really know. Not sure what the right thing to do is regarding copyright blocks and dates.